### PR TITLE
Estimated time left

### PR DESCRIPTION
**Pull request by kewde**
Fix estimated time left
Average the number out, prevent ridiculous hopping

Someone changed some stuff without knowing what it did, it was quite hard to figure this out.

### CODE REVIEW

**Comment by kewde** [comment]
Testing instructions:

```
user@host:~/.particl$ mv testnet old-testnet
user@host:~/.particl$ rm -rf testnet/
user@host:~/.particl$ mkdir testnet
user@host:~/.particl$ cp old-testnet/peers.dat testnet/peers.dat
user@host:~/.particl$ cp old-testnet/wallet.dat testnet/wallet.dat
```

Start Electron GUI.

**Comment by rynomster** [comment]
I changed it because it was using new Date() always.. was never using mediantime, then I added lastblock so that it would constantly updated based on last block.. median time will now only update every 2 seconds, so you won't have a smooth update on the percentage bar.

**Comment by kewde** [comment]
> The sync screen changes every few seconds instead of every time a block comes in

There's no ZMQ, so it's always with a polling time, I can increase the polling time.

File: src/app/core/rpc/rpc-state.class.ts [hunk 1]
```diff
@@ -13,30 +13,12 @@ export class RPCStateClass {
...
     this.rpc.registerStateCall('getstakinginfo', 10000);
 
-    this.lastBlockTimeState();
     this.blockLoop();
     this.walletLockedState();
     this.coldStakeHook();
     this.initWalletState();
   }
 
-  private lastBlockTimeState() {
```
Comment (rynomster) [inline src/app/core/rpc/rpc-state.class.ts]: This code calls getblockchaininfo everytime it needs to... you can put this code back and the blocks will update as required

**Comment by kewde** [comment]
#214

**Pull request merged**